docs/diagnostics/PX1015.md

```diff
@@ -8,7 +8,9 @@ This document describes the PX1015 diagnostic.
 | PX1015 | For a BQL statement that contains parameters, the number of arguments of a `Select` method is different from the number of parameters. | Warning (Level 1: Significant) | Unavailable | 
 
 ## Diagnostic Description
-For BQL statements that contain `Required` and `Optional` parameters, a `Select` method must have a number of arguments that is equal to or greater than the number of `Required` parameters in the BQL statement. 
+For BQL statements that contain `Required` and `Optional` parameters, a `Select` method must have a number of arguments that satisfies both of the following requirements:
+ - The number of arguments is equal to or greater than the number of `Required` parameters in the BQL statement.
+ - The number of arguments is equal to or less than the number of `Required` and `Optional` parameters in the BQL statement.
```

Comment:
@k-s-popova The number of arguments in this case is greater than the number of `Required` parameters or equal to it and less than the number of `Required` + `Optional` parameters or equal to it:
`RequredCount <= SelectArgsCount <= RequiredCount + OptionalCount`